cli/src/cluster_query.rs

```diff
@@ -1817,11 +1865,13 @@ pub fn process_show_stakes(
                         });
                     }
                 }
-                StakeState::Stake(_, stake) => {
-                    if vote_account_pubkeys.is_none()
+                StakeState::Stake(meta, stake) => {
+                    if (vote_account_pubkeys.is_none()
                         || vote_account_pubkeys
                             .unwrap()
-                            .contains(&stake.delegation.voter_pubkey)
+                            .contains(&stake.delegation.voter_pubkey))
+                        && (withdraw_authority_pubkey.is_none()
+                            || (*withdraw_authority_pubkey.unwrap()) == meta.authorized.withdrawer)
```

Comment:
This will be covered by the RPC filter, so it can be removed as well.

Reply:
Yes i agree but it was done for vote_account_pubkeys which seems weird as it also be covered by RPC filter.

Reply:
Ahh yes I understand.